mail-client/alot/alot-0.8.1.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
```

Comment:
Python 3.4 is deprecated
Python 3.7 is supported, as stated [here](https://github.com/pazz/alot/blob/master/setup.py#L26)

Reply:
If I add 3.7 repoman throw me a lot of error, as 3.7 is still unstable in the tree,
it would need to upgrade those ebuild becuase they do not support 3.7 actually (dev-python/urwidtrees, dev-python/configobj).
```
$ sudo repoman scan -d

RepoMan scours the neighborhood...
  dependency.bad [fatal]        14
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',                                           
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']                                            
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0/desktop)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',                                           
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']                                            
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0/desktop/gnome)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',                                           
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']                                            
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0/desktop/gnome/systemd)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',                                           
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']                                            
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0/desktop/plasma)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',                                           
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']                                            
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0/desktop/plasma/systemd)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',                                           
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']                                            
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0/developer)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',                                           
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']                                            
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0/hardened)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',                                           
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']                                            
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0/hardened/selinux)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',                                           
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']                                            
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0/no-multilib)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',                                           
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']                                            
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0/no-multilib/hardened)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',                                           
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']                                            
   mail-client/alot/alot-0.8.1.ebuild: RDEPEND: ~amd64(default/linux/amd64/17.0/no-multilib/hardened/selinux)
[     '>=dev-python/configobj-4.7.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]',
      '>=dev-python/urwidtrees-1.0[python_targets_python3_5(-)?,python_targets_python3_6(-)?,python_targets_python3_7(-)?,-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-)]']
  dependency.badindev           15
Please fix these important QA issues first.
RepoMan sez: "Make your QA payment on time and you'll never see the likes of me."
```

Reply:
I'm sorry for misleading you.
`dev-python/urwidtrees` and `dev-python/configobj` aren't Python3.7 enabled still.